spec/models/lineup_positions_spec.rb

```diff
@@ -8,69 +8,82 @@
   it "has no duplicate players"
   it "has no duplicate positions"
 
-  describe "position change" do
-    let!(:player1) { create(:player, win: 2, loss: 1) }
-    let!(:player2) { create(:player, win: 1, loss: 2) }
-    let!(:player3) { create(:player, win: 3, loss: 0) }
-    let!(:player4) { create(:player, win: 2, loss: 1) }
-    let!(:no_match_p1) { create(:player) }
-    let!(:no_match_p2) { create(:player) }
-    let!(:no_match_p3) { create(:player) }
-
+  describe "changing position" do
+    let!(:player1) { create(:player) }
+    let!(:player2) { create(:player) }
+    let!(:player3) { create(:player) }
     let!(:lineup1)  { create(:player_lineup) }
-    let!(:lineup2)  { create(:player_lineup) }
-
     let!(:position1) { create(:lineup_position, position: 1, player: player1, player_lineup: lineup1) }
     let!(:position2) { create(:lineup_position, position: 2, player: player2, player_lineup: lineup1) }
     let!(:position3) { create(:lineup_position, position: 3, player: player3, player_lineup: lineup1) }
-    let!(:position4) { create(:lineup_position, position: 4, player: player4, player_lineup: lineup1) }
-    let!(:no_match_pos1) { create(:lineup_position, position: 1, player: no_match_p1, player_lineup: lineup2) }
-    let!(:no_match_pos2) { create(:lineup_position, position: 2, player: no_match_p2, player_lineup: lineup2) }
-    let!(:no_match_pos3) { create(:lineup_position, position: 3, player: no_match_p3, player_lineup: lineup2) }
 
-    context "without playing matches" do
+    context "with no matches played" do
       it "moves without restriction" do
-        no_match_pos3.position = 1
-        expect(no_match_pos3).to be_valid
+        position3.position = 1
+        expect(position3).to be_valid
```

Comment:
The rewrite won't work because as you alluded to `position3` is valid to begin with. The spec is meant to express that, without matches played, `position3` may shift multiple `#position`s and still remain valid. As you said, it's valid regardless of the value of `#position` Let me know if you've thoughts on how I might express this better.